lib/manageiq/release/internationalization.rb

```diff
@@ -0,0 +1,199 @@
+module ManageIQ
+  module Release
+    class Internationalization
+
+      attr_reader :branch, :dry_run
+
+      def initialize(branch: nil, dry_run: true)
+        @branch  = branch || 'master'
+        @dry_run = dry_run
+      end
+
+      def update_message_catalogs
+        klasses = []
+        klasses << ManageIQ
+        klasses << ManageIQ_ServiceUI
+
+        klasses.each do |klass|
+          klass.new(branch: branch, dry_run: dry_run).update_message_catalog
+        end
+      end
+
+      class Helper
+        def execute(command, description = nil)
+          description ||= "*** Running `#{command}` ***"
+          puts description
+          system(command)
```

Comment:
If the command fails should we bail?  If so, we have a `system!` method that could be used which will `exit 1` and output the error.